src/fu-usb-backend.c

```diff
@@ -20,6 +20,29 @@ struct _FuUsbBackend {
 
 G_DEFINE_TYPE(FuUsbBackend, fu_usb_backend, FU_TYPE_BACKEND)
 
+#define FU_USB_BACKEND_POLL_INTERVAL_DEFAULT	 1000 /* ms */
+#define FU_USB_BACKEND_POLL_INTERVAL_WAIT_REPLUG 5 /* ms */
+
+static void
+fu_usb_backend_device_notify_flags_cb(FuDevice *device, GParamSpec *pspec, FuBackend *backend)
+{
+#if G_USB_CHECK_VERSION(0, 3, 10)
+	FuUsbBackend *self = FU_USB_BACKEND(backend);
+
+	/* if waiting for a disconnect, set win32 to poll insanely fast -- and set it
+	 * back to the default when the device removal was detected */
+	if (fu_device_has_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG)) {
+		g_debug("setting USB poll interval to %ums to detect replug",
+			(guint)FU_USB_BACKEND_POLL_INTERVAL_WAIT_REPLUG);
+		g_usb_context_set_hotplug_poll_interval(self->usb_ctx,
```

Comment:
Shouldn't this only run on win32?

Reply:
it's a NOP on !WIN32, but maybe there's a valid argument to not do the `g_signal_connect` on !WIN32

Reply:
Yeah sounds good.

Reply:
Done.